src/mainboard/sunxi/nezha/fixed-dtfs.dts

```diff
@@ -0,0 +1,46 @@
+/dts-v1/;
+
+/ {
+    #address-cells = <1>;
+    #size-cells = <1>;
+
+    flash-info {
+        compatible = "ore-flashinfo";
+        board-name = "HiFive Unleashed";
+        category = "SiFive";
+        board-url = "https://www.sifive.com/boards/hifive-unleashed";
+        areas {
+            area@0 {
+                description = "Boot Blob and Romstage";
+                size = <0x80000>; // 512KiB
+                file = "$(TARGET_DIR)/bootblob.bin";
+            };
+            area@1 {
+                description = "Fixed DTFS";
+                size = <0x80000>; // 512KiB
+                file = "$(TARGET_DIR)/fixed-dtfs.dtb";
+            };
+            area@2 {
```

Comment:
TODO: the tool should be able to size areas dynamically. The fixed sizes are because we don't yet walk to fdt nodes ... not your problem, just mentioning it.